### PR TITLE
Exam Types - Exam Duration with Minutes

### PR DESCRIPTION
**Pull request by akroon3r**
Client testing found that pesticide exams had durations that weren't round number of hours (ie// 1h30m, 1h45m). To accomodate for this, a number_of_minutes field was added to the exam_type model/schema and then added to booking durations in the calendar, booking-modal and edit-booking-modal vue files. This was tested with the client over zoom to ensure that there was no confusion on how exams were created and handled.

### CODE REVIEW

**Comment by sjrumsby** [comment]
This will need to be updated with the new schema changes from the pesticide PR.

**Comment by akroon3r** [comment]
Fixed, pushed @sjrumsby

File: api/app/admin/examtype.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def get_query(self):
         'exam_type_name',
         'exam_color',
         'number_of_hours',
+        'number_of_minuts',
```
Comment (sjrumsby) [inline api/app/admin/examtype.py]: typo

Comment (akroon3r) [reply api/app/admin/examtype.py]: fixed, pushed

**Pull request merged**